src/pointer-analysis/goto_program_dereference.h

```diff
@@ -63,12 +63,14 @@ class goto_program_dereferencet:protected dereference_callbackt
   value_setst &value_sets;
   value_set_dereferencet dereference;
 
+  DEPRECATED("Unused")
```

Comment:
Symex (which now has been moved to a different repository) might be using these. @kroening?